packages/destination-actions/src/destinations/fullstory/vars.ts

```diff
@@ -107,11 +111,39 @@ const typeSuffixPropertyName = (name: string, value: unknown) => {
   return name
 }
 
+type PropertyNameTransformation = (_: string) => string
```

Comment:
big fan of these transformations! 😍  We might be able to leverage a more generic transformation type such as `type Transformation<T> = (s: T) => T` if we see this pattern more.